.github/workflows/test-get.yaml

```diff
@@ -2,8 +2,7 @@
 name: test-get
 
 on:
-  # Temporarily deactivating while method is not implemented
-  workflow_dispatch:
+  - push
```

Comment:
This workflow never use the `get` method

Reply:
what? yes it does https://github.com/scality/action-artifacts/blob/development/2.0/.github/workflows/test-get.yaml#L50

Reply:
I think I know what brought you to this conclusion, the steps were named as "Artifacts upload" while they were using the get method, which can be confusing, I just renamed the steps. The code/tests remains the same